ssconvert.rb

```diff
@@ -0,0 +1,36 @@
+require 'roo'
+
+def output_filename(sheet)
+  "inputs/efile_COAK_2016_#{sheet}.csv"
+end
+
+xlsx = Roo::Spreadsheet.open(ARGV[0])
+
+$stderr.puts "Converting #{ARGV[0]} to a series of CSV files..."
+
+xlsx.sheets.each do |sheet|
+  out = output_filename(sheet)
+
+  $stderr.puts "  -> #{out}"
+  file = xlsx.sheet(sheet).each_row_streaming.each_entry
+
+  headers = file.next
+
+  CSV.open(out, 'wb') do |csv|
+    csv << headers
+
+    loop do
+      row = file.next
+
+      # HACK: the datetimes seem to have a weird formatting string
+      # ("mm/dd/yyy") in the XLSX spreadsheet which has an extra "y" at the end.
+      row.each do |cell|
+        if cell.type == :date
+          cell.instance_variable_set(:@format, "yyyy-mm-dd")
```

Comment:
you say its yyy but then put yyyy here.  Perhaps the comment needs to be clarified?